pkg/neorpc/filters.go

```diff
@@ -49,6 +54,16 @@ type (
 	}
 )
 
+// SubscriptionFilter is an interface for all subscription filters.
+type SubscriptionFilter interface {
+	// IsValid checks whether the filter is valid and returns
+	// a specific [ErrInvalidSubscriptionFilter] error if not.
+	IsValid() error
+}
+
+// neorpc.ErrInvalidSubscriptionFilter is returned when the subscription filter is invalid.
+var ErrInvalidSubscriptionFilter = errors.New("invalid subscription filter")
```

Comment:
Exported comment should start with `ErrInvalidSubscriptionFilter`, you already in the `neorpc` package.

Reply:
@AliceInHunterland, fix this, please.

Reply:
https://github.com/nspcc-dev/neo-go/pull/3258#discussion_r1432521382 What does than little more specific mean?

Reply:
fixed